reference/Articles-Sync-API.yaml

```
@@ -0,0 +1,98 @@
+x-stoplight:
+  docs:
+    showModels: true
+x-internal: true
+openapi: 3.1.0
+info:
+  title: Articles Sync API Reference
+  version: "1.0"
+  contact:
+    name: ''
+    email: ''
+  summary: Internal service for syncing articles between CloudMatrix and Playback Store
+  tags:
+  - name: Articles Sync
+servers:
+  - url: "https://cloudmatrix-articles-service.streamamg.com"
+paths:
+ '/articles/sync':
+   post:
+      summary: Articles sync
+      tags:
+        - Sync
+      description: 'Synchronizes articles from CM to Playback store'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              description: ''
+              type: object
+              properties:
+                site:
+                  type: string
+                  description: 'Specifies the site whose articles are to be synchronized'
```

minor: make requestBody and site in it required

```suggestion
      requestBody:
        required: true
        content:
          application/json:
            schema:
              description: ''
              type: object
              properties:
                site:
                  type: string
                  description: 'Specifies the site whose articles are to be synchronized'
              required:
                - site
```